tools/benchmarks/llm_eval_harness/meta_eval/prepare_meta_eval.py

```diff
@@ -134,6 +134,18 @@ def change_yaml(args, base_name):
                         "WORK_DIR", str(yaml_dir)
                     )
                 )
+    # 3.2 evals dataset has a differents set of evals
+    if args.evals_dataset in [
+        "meta-llama/Llama-3.2-1B-evals",
+    ]:
+        # Change meta_pretrain.yaml to load in supported evals
+        with open(args.template_dir + "/meta_pretrain.yaml", "r") as yaml_file:
+            meta_pretrain = yaml.safe_load(yaml_file)
+            meta_pretrain["task"] = ["meta_mmlu"]
+        
+        with open(args.work_dir + "/meta_pretrain.yaml", "w") as yaml_file:
+            yaml.dump(meta_pretrain, yaml_file)
+
```

Comment:
Not a sustainable approach in the long-term, but decided against a refactor at this stage; We only have two cases for meta_pretrain atm - 3.1 and 3.2. 1 or 2 more cases and it'd be a good time to refactor imo.

But lmk if you'd like me to refactor. One approach is to create the meta_pretrain.yaml file from scratch:

```python
meta_pretrain = {
    group: "meta_pretrain"
}
if is_3_1_dataset(args.evals_dataset):
    meta_pretrain["task"] = ["meta_mmlu"]
elif is_3_2_dataset(args.evals_dataset):
    meta_pretrain["task"] = ["meta_bbh", "meta_mmlu_pro_pretrain"]

with open(args.work_dir + "/meta_pretrain.yaml", "w") as yaml_file:
    yaml.dump(meta_pretrain, yaml_file)
```